.env.template

```diff
@@ -12,7 +12,17 @@ PLAID_ENV=sandbox
 
 # If you want to test the app using a payment processor, set PROCESSOR to 'true'
 
+
+# The processor we use for testing is Dwolla.  If you would like to test with Dwolla, set IS_PROCESSOR to true
+# You will need to open a sandbox account at https://developers.dwolla.com/guides/sandbox. Fill in the temporary
+# DWOLLA_ACCESS_TOKEN that can be created on your dashboard at https://dashboard-sandbox.dwolla.com/applications-legacy.
+# This is a temporary token that is good for one hour before you need to create another one.  Also enter your
+# DWOLLA_MASTER_ACCOUNT_ID from your Superhero Savings Bank master account on your dashboard at
+# https://dashboard-sandbox.dwolla.com/account/funding-sources
+
 IS_PROCESSOR=false
+DWOLLA_ACCESS_TOKEN=
+DWOLLA_MASTER_ACCOUNT_ID=
```

Comment:
oh btw don't forget `code tags` around code snippets or names of variables / parameters!